### PR TITLE
Release 0.5.0

### PR DESCRIPTION
**Pull request by lorenzo-cavazzi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 # Changes
 
+## 0.5.0 (released 2019-05-22)
+
+* Refactored Notebooks page and Notebook Servers tab in project page [#415](https://github.com/SwissDataScienceCenter/renku-ui/issues/415), [#425](https://github.com/SwissDataScienceCenter/renku-ui/pull/425)
```
Comment (ciyer) [inline CHANGELOG.md]: `New top-level Notebooks and project Notebook Servers interfaces with more features.`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+* Refactored Files tab in project page [#191](https://github.com/SwissDataScienceCenter/renku-ui/issues/191), [#400](https://github.com/SwissDataScienceCenter/renku-ui/issues/400), [#420](https://github.com/SwissDataScienceCenter/renku-ui/pull/420), [#435](https://github.com/SwissDataScienceCenter/renku-ui/pull/435), [#437](https://github.com/SwissDataScienceCenter/renku-ui/pull/437), [#447](https://github.com/SwissDataScienceCenter/renku-ui/pull/447)
```
Comment (ciyer) [inline CHANGELOG.md]: `Full-featured file browsing with a project.`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,22 @@
...
+* Implemented lazy loading for Files tab in project page [#448](https://github.com/SwissDataScienceCenter/renku-ui/issues/448), [#455](https://github.com/SwissDataScienceCenter/renku-ui/pull/455)
```
Comment (ciyer) [inline CHANGELOG.md]: Add this to the previous line. No need for this PR to have its own summary.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,22 @@
...
+* Refactored lineage pages to use the graph service [#362](https://github.com/SwissDataScienceCenter/renku-ui/issues/362), [#392](https://github.com/SwissDataScienceCenter/renku-ui/pull/392)
```
Comment (ciyer) [inline CHANGELOG.md]: `Lineage display uses information provided by new Knowledge Graph service.`

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,22 @@
...
+* Added graph indexing status indications while browsing lineage pages [#451](https://github.com/SwissDataScienceCenter/renku-ui/issues/451), [#464](https://github.com/SwissDataScienceCenter/renku-ui/pull/464)
```
Comment (ciyer) [inline CHANGELOG.md]: Add PRs to previous line. Does not need its own summary.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,22 @@
...
+* Added knowledge graph inspection to project pages [#369](https://github.com/SwissDataScienceCenter/renku-ui/issues/369), [#374](https://github.com/SwissDataScienceCenter/renku-ui/pull/374), [#453](https://github.com/SwissDataScienceCenter/renku-ui/pull/453)
```
Comment (ciyer) [inline CHANGELOG.md]: `Added tools for configuration and status of integration with new Knowledge Graph service.`

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,22 @@
...
+* Added knowledge graph informations to new project page [#417](https://github.com/SwissDataScienceCenter/renku-ui/issues/417), [#419](https://github.com/SwissDataScienceCenter/renku-ui/pull/419)
```
Comment (ciyer) [inline CHANGELOG.md]: Add to above line.

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,22 @@
...
+* Improved landing page [#383](https://github.com/SwissDataScienceCenter/renku-ui/issues/383), [#398](https://github.com/SwissDataScienceCenter/renku-ui/issues/398), [#423](https://github.com/SwissDataScienceCenter/renku-ui/pull/423)
+* Fixed project page for non-accessible projects [#389](https://github.com/SwissDataScienceCenter/renku-ui/issues/389), [#406](https://github.com/SwissDataScienceCenter/renku-ui/issues/406), [#421](https://github.com/SwissDataScienceCenter/renku-ui/pull/421)
+* Fixed redirect after login [#438](https://github.com/SwissDataScienceCenter/renku-ui/pull/438)
```
Comment (ciyer) [inline CHANGELOG.md]: `Redirect user back to currently-displayed page after login.`

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,22 @@
...
+* Fixed bug for anonymous users [#422](https://github.com/SwissDataScienceCenter/renku-ui/pull/422)
```
Comment (ciyer) [inline CHANGELOG.md]: `Fixed crash when previewing a notebook as an anonymous user.`

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,19 @@
 # Changes
 
+## 0.5.0 (released 2019-05-22)
+
+* New top-level Notebooks and project Notebook Servers interfaces with more features [#415](https://github.com/SwissDataScienceCenter/renku-ui/issues/415), [#425](https://github.com/SwissDataScienceCenter/renku-ui/pull/425)
+* Full-featured file browsing with a project [#191](https://github.com/SwissDataScienceCenter/renku-ui/issues/191), [#400](https://github.com/SwissDataScienceCenter/renku-ui/issues/400), [#420](https://github.com/SwissDataScienceCenter/renku-ui/pull/420), [#435](https://github.com/SwissDataScienceCenter/renku-ui/pull/435), [#437](https://github.com/SwissDataScienceCenter/renku-ui/pull/437), [#447](https://github.com/SwissDataScienceCenter/renku-ui/pull/447), [#455](https://github.com/SwissDataScienceCenter/renku-ui/pull/455)
```
Comment (ciyer) [inline CHANGELOG.md]: `with` -> `within`.

Comment (lorenzo-cavazzi) [reply CHANGELOG.md]: Corrected

**Pull request merged**